pkg/v1/providers/ytt/04_user_customizations/README.md

```diff
@@ -1,3 +1,3 @@
 # User Customizations
 
-ytt traverses directories in alphabetical order and overwrites duplicate settings. You can create template files here to contain configurations that take precedence over any in lower-numbered ytt subdirectories.
\ No newline at end of file
+ytt traverses directories in alphabetical order and overwrites duplicate settings. You can create template files here to contain configurations that take precedence over any in lower-numbered ytt subdirectories.
```

Comment:
*wiping my eyes* where is the diff :-) ?!

Reply:
oh the CI linting was complaining there is no newline character. So I added a `\n`